### ISSUE TITLE
Blood PR Causes Surgery Infections That Shouldn't Happen

### ISSUE THREAD

**Issue by Elasmobranchii** [ParadiseSS13/Paradise#26158]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1639

### Issue Description

When blood splashes onto hands during a patient's surgery, the blood can then cause infections if the doctor continues to operate on the patient. There is something very wrong with how germ levels are currently handled with gloves.

### What did you expect to happen?

Germs to properly accumulate on gloves, causing infection if a doctor switches patients without washing in-between.

### What happened instead?

Instead, there is a very high chance of any surgery on a patient causing an infection without use of drapes or spaceacillin, even if a doctor starts surgery with clean hands and clean tools.

### Why is this bad/What are the consequences?

This is an important issue because it's yet another unintended consequence of #25730 which wasn't intended to affect Medical. While this bug can largely be mitigated by drapes, it can still add unnecessary strain on spaceacillin stock, and more strain on medical staff.

### Steps to reproduce the issue.

-Start surgery with gloves (even sterile ones)
-Operate on a patient
-Get splashed with blood
-Continue Surgery
-Observe Infection

### When did the problem start happening?

After #25730 was merged. Has persisted through its various fixes.

**Comment by Fordoxia** [comment]
I could have sworn that bloody hands from surgery has been a thing since forever.

**Comment by Elasmobranchii** [comment]
It is. But what isn't since forever is surgery always causing infections. The only PR I could point to as the culprit is the one I linked...Unless it did get fixed when infinite spread got fixed, and another PR broke it, I'll admit I don't know enough about code to have done a deep dive on it.

if it helps, I have a master build from 2 days prior to the merge I tested on, no infection issue.

**Comment by Fordoxia** [comment]
> It is. But what isn't since forever is surgery always causing infections. The only PR I could point to as the culprit is the one I linked...Unless it did get fixed when infinite spread got fixed, and another PR broke it, I'll admit I don't know enough about code to have done a deep dive on it.
> 
> if it helps, I have a master build from 2 days prior to the merge I tested on, no infection issue.

There's another PR that nerfed the cautery (which previously deleted all germs) #25959.

**Comment by HMBGERDO** [comment]
blood was on your hands before my PR too
my PR does not affect surgery germs or blood spreading during surgery so...

**Comment by silverplatedelta** [comment]
Fixed in #25959.
